plugins/hls-tactics-plugin/src/Wingman/Types.hs

```diff
@@ -117,6 +117,9 @@ instance Show Class where
 instance Show (HsExpr GhcPs) where
   show  = unsafeRender
 
+instance Show (HsExpr GhcTc) where
+  show  = unsafeRender
+
 instance Show (HsDecl GhcPs) where
   show  = unsafeRender
 
```

Comment:
Please consider moving these to the Orphans module

Reply:
Happy to --- followup PR ok?